src/consensus/zmq_driver.rs

```diff
@@ -352,10 +352,8 @@ fn handle_update(
         }
         CONSENSUS_NOTIFY_ENGINE_DEACTIVATED => Update::Shutdown,
         unexpected => {
-            return Err(Error::ReceiveError(format!(
-                "Received unexpected message type: {:?}",
-                unexpected
-            )));
+            error!("Received unexpected message type: {:?}", unexpected);
```

Comment:
- Change the log message from `error` to `warn`
- Either change "Received" to "Ignoring" or add "; ignoring" to the end of the log message.

Reply:
Updated